nvflare/fuel/hci/client/cli.py

```diff
@@ -112,6 +114,10 @@ def __init__(
 
         self._get_login_creds()
 
+        event_handlers = [self]
+        if handlers:
+            event_handlers.extend(handlers)
```

Comment:
better to check handlers are instances of EventHandler?

Reply:
Already checked in parsing the config.